GitUI/ProcessOutputTimer.cs

```diff
@@ -19,21 +19,17 @@ public ProcessOutputTimer(DoOutputCallback doOutput)
             _timer.Tick += _timer_Elapsed;
         }
 
-        public void Start(int interval)
+        public void Start(int interval = 600)
```

Comment:
💭 I prefer using `TimeSpan` over `int`, which currently requires `TimeSpan?` to use with a default argument. The limitation may be lifted as part of the work on dotnet/csharplang#187.

Reply:
Agree. In #4518 I changed the `AsyncLoader.Delay` from `int` to `TimeSpan`. At the very least the name or docs should indicate the units. Early .NET framework code was bad at this in places (case in point, the WinForms `Timer` class used herein).